### PR TITLE
Updates main README to include information about each sample.

### PR DESCRIPTION
**Pull request by codingjeremy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+
+Read below for a description of each sample.
+
+
+Samples
+----------
+
+* **[AlwaysOn](https://github.com/android/wear-os-samples/tree/main/AlwaysOn)**(Java) - Demonstrates keeping the app visible in ambient mode. Note: While this is valuable for some specific use cases, most use cases won't need this, because when the watch goes into ambient mode (and shows watch face), if the user interactes with the watch again within several minutes, it will bring the app back up when it transitions out of ambient mode.
+
+* **[DataLayer](https://github.com/android/wear-os-samples/tree/main/DataLayer)**(Java) - Demostrates communicating via the data layer between the watch and the phone.
```
Comment (JolandaVerhoef) [inline README.md:14]: Typo: Demostrates should be Demonstrates

File: README.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+
+* **[RuntimePermissionsWear](https://github.com/android/wear-os-samples/tree/main/RuntimePermissionsWear)**(Java) - Demostrates requesting permissions not only on a Wear OS device and a mobile device, but also across devices, e.g., request permissions on a mobile device from your wear device and the other way around. Note: It does require you having both the Wear and Mobile version of the app installed on both devices.
```
Comment (JolandaVerhoef) [inline README.md:16]: Typo: Demonstrates

File: README.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+
+* **[WatchFace](https://github.com/android/wear-os-samples/tree/main/WatchFace)**(Java) - Shows a full implementation of multiple watch faces, complications, and settings (edit watch faces appearance). Also includes implementation of data providers, so developers can share their data to be displayed in complications on any watch face (without writing a watch face). Data providers are quite easy to implement and allow developers to own part of the screen real estate of a watch face.
```
Comment (JolandaVerhoef) [inline README.md:18]: The last sentence here seems a bit out of context. Maybe you can leave it out.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+
+* **[WearAccessibilityApp](https://github.com/android/wear-os-samples/tree/main/WearAccessibilityApp)**(Java) - Sample demonstrates how to include accessibility support for your wearable app.
+
+* **[WearComplicationProvidersTestSuite](https://github.com/android/wear-os-samples/tree/main/WearComplicationProvidersTestSuite)**(Java) - If you are writing a watch face with complications, this app gives you a full suite of data providers to test against your implimentation of complications to make sure it looks good.
```
Comment (JolandaVerhoef) [inline README.md:22]: Typo: Implementation

**Review by JolandaVerhoef — COMMENTED** [review]
Some comments:
- You can use relative links to the samples, see https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files
- Would it make sense to link to the relevant developer.android.com documentation pages?

**Review by codingjeremy — COMMENTED** [review]
Fixed.

**Review by sanalhapishane — APPROVED** [review]
Eline sağlık 👈

**Review by sanalhapishane — APPROVED** [review]
👍

**Review by JolandaVerhoef — APPROVED** [review]

**Comment by codingjeremy** [comment]
@JolandaVerhoef I converted all links to relative paths and added guides. I'm carrying over your approval. Thanks!

**Pull request merged**